packages/component-library/scripts/icons/downloadIcons.mjs

```diff
@@ -0,0 +1,56 @@
+import fetch from 'cross-fetch';
+import fse from 'fs-extra';
+import path from 'path';
+import Queue from './waterfall/Queue.mjs';
+import sleep from './waterfall/sleep.mjs';
+import retry from './waterfall/retry.mjs';
+
+async function downloadIcon(icon) {
+  console.log(`downloadIcon ${icon.index}: ${icon.name}`);
+
+  const response = await fetch(
+    `https://fonts.gstatic.com/s/i/materialiconsoutlined/${icon.name}/v${icon.version}/24px.svg`
+  );
+  if (response.status !== 200) {
+    throw new Error(`status ${response.status}`);
+  }
+  const SVG = await response.text();
+  await fse.writeFile(
+    path.join(
+      'scripts',
+      'icons',
+      'material-icons',
+      `${icon.name}_outlined_24px.svg`
+    ),
+    SVG
+  );
+}
+
+async function run() {
+  try {
+    await fse.emptyDir(path.join('scripts', 'icons', 'material-icons'));
+    const response = await fetch('https://fonts.google.com/metadata/icons');
```

Comment:
Do we filter to download only outline icons? Or do we download all icons?

Reply:
Some icons do not exist. 
I think the issue is coming from the fact that, at this line of code, you download a JSON with all icons.
Later, when downloading the icons, you use the URL `https://fonts.gstatic.com/s/i/materialiconsoutlined/${icon.name}/v${icon.version}/24px.svg` which refers to the outline icons only.
![image](https://user-images.githubusercontent.com/55230735/155992608-3fae6e83-289a-46cd-8575-4d74b51a585f.png)

Reply:
Or maybe I'm wrong because obviously, the icon from my screenshot above exists: https://fonts.gstatic.com/s/i/materialiconsoutlined/swap_horizontal_circle/v12/24px.svg